k8s/sample-pv-yamls/blockdeviceclaim.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: openebs.io/v1alpha1
+kind: BlockDeviceClaim
+metadata:
+  name: test-blockdeviceclaim
+  namespace: openebs
+spec:
+  driveType: ""
```

Comment:
Should this be blank? What's the default?

Reply:
Added comments to this file